### ISSUE TITLE
Issue on loading model: std::runtime_error at memory location

### ISSUE THREAD

**Issue by latinusveridas** [Dobiasd/frugally-deep#306]
Hello,

I'm facing to an issue when I try to load my model in C++, which seems relatively straight (Conv2D, BatchNormalization, relu activation etc.  ), I've got an error message at load_model step:

```
#define FDEEP_FLOAT_TYPE double
#include "fdeep/fdeep.hpp"

using namespace std;

int main()
{
	string model_fp = "C:/Users/x/Desktop/25_09_works_on_frugally_deep/my_model.json";
	const auto model = fdeep::load_model(model_fp); <== Error happens here

	return 0;
}
```

The error message:
`Exception thrown at 0x00007FFC3A034F99 in frugadeep_testing.exe: Microsoft C++ exception: std::runtime_error at memory location 0x00000068F03ED650.
Unhandled exception at 0x00007FFC3A034F99 in frugadeep_testing.exe: Microsoft C++ exception: std::runtime_error at memory location 0x00000068F03ED650.`

You can find the converted model at this address, as the file is to big to be included here: https://file.io/51kwGosVzhHP 

Please find additional info below:

- Tensorflow version 2.6
- Python: 3.8.10
- Visual Studio 2019, set as Release + x64
- frugally-deep installed through vcpkg

Please let me know if you need additional info
Thank you for your help,
Latinus

**Comment by Dobiasd** [comment]
Hi,

this uncaught exception does have a message attached (as do most exceptions).
I recommend learning how to view them in your environment.

Anyway, I've run `fdeep::load_model` with your `.json` file, and the output is the following:

```
Loading json ... done. elapsed time: 0.230807 s
Building model ... terminate called after throwing an instance of 'std::runtime_error'
  what():  unknown layer type: Conv2DTranspose
```

For an explanation on why `Conv2DTranspose` is not supported, as well as a suggested alternative approach, please see ["Why are Conv2DTranspose layers not supported?" in the FAQ](https://github.com/Dobiasd/frugally-deep/blob/master/FAQ.md#why-are-conv2dtranspose-layers-not-supported). :slightly_smiling_face:

**Comment by latinusveridas** [comment]
Hi Dobiasd,

Thank very much for this comprehensive answer, I forgot that I have Conv2DTranspose layers in my model, I'll change that. 

Have a nice day,
Latinus
